### ISSUE TITLE
Feature Request: Open in float

### ISSUE THREAD

**Issue by tami5** [nvim-telescope/telescope.nvim#365]
It would be cool if there is an action and mapping to open an entry in float

**Comment by sunjon** [comment]
what "properties" would the window have once it's no longer a telescope window? I guess this is related to persistent windows.

**Comment by tami5** [comment]
@sunjon Yes, I believe  it is related to #310 . For properties, it should simply inherits the same window properties that of the telescope instance .

**Comment by TC72** [comment]
I've just added a PR for this #1299
